cmd/minikube/cmd/start.go

```diff
@@ -1779,3 +1781,61 @@ func exitGuestProvision(err error) {
 	}
 	exit.Error(reason.GuestProvision, "error provisioning guest", err)
 }
+
+func startNerdctld() {
+	// create a temporary directory
+	tmpDirName, err := os.MkdirTemp("/tmp", "minikube-nerdctld")
```

Comment:
lets leave this empty string so it will use default and it would work on windows and eveyrwhere !
```tmpDirName, err := os.MkdirTemp("", "minikube-nerdctld")```

Reply:
let ensure to delete and clean up after ourselves

```defer os.RemoveAll(tmpDirName)```